packages/core/src/constants/index.ts

```diff
@@ -1 +1,5 @@
 export const protectedAppSignInCallbackUrl = 'sign-in-callback';
+/** The default lifetime of subject tokens (in seconds) */
+export const subjectTokenExpiresIn = 600;
+/** The prefix for subject tokens */
+export const subjectTokenPrefix = 'st_';
```

Comment:
wondering if we need to indicate "token" here. how about:

```suggestion
export const subjectTokenPrefix = 'sub_';
```